cves/CVE-2012-4520.yml

```diff
@@ -1,300 +1,323 @@
-    why you come to that conclusion.
-  answer_note:
-  answer:
-subsystem:
+      Was the feature impacted by this vulnerability about internationalization
+      (i18n)? An internationalization feature is one that enables people from all
+      over the world to use the system. This includes translations, locales,
+      typography, unicode, or various other features.
+      
+      Answer should be boolean. Write a note about how you came to the conclusions
+      you did.
+interesting_commits: 
+  commits: 
+    - 
+      commit: 0714b0f39033a51e447bc1b3b03ffee6f9228105
+      note: "This commit affects parsing of values in HTTP Header. This is not explicitly related to passwords but it could maybe have an effect on the way an entered value was interpreted by the Header, which is where the vulnerability was.\n"
+    - 
+      commit: ~
+      note: ~
   question: |
-    What subsystems was the mistake in?
-
-    Most systems don't have a formal list of their subsystems, but you can
-    usually infer them from path names, bug report tags, or other key words
-    used. A single source file is not what we mean by a subsystem. In Django,
-    the "Component" field on the bug report is useful. But there may be other
-    subsystems involved.
-
-    Your subsystem name(s) should not have any dots or slashes in them. Only
-    alphanumerics, whitespace, _, - and @.Feel free to add multiple using a YAML
-    array.
-
-    In the answer field, explain where you saw these words.
-    In the name field, a subsystem name (or an array of names)
-
-    e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
-interesting_commits:
+      Are there any interesting commits between your VCC(s) and fix(es)?
+      
+      Write a brief (under 100 words) description of why you think this commit was
+      interesting in light of the lessons learned from this vulnerability. Any
+      emerging themes?
+ipc: 
+  answer: true
+  note: "messages inputted into password value can pass to files like configuration.\n"
   question: |
-    Are there any interesting commits between your VCC(s) and fix(es)?
-
-    Write a brief (under 100 words) description of why you think this commit was
-    interesting in light of the lessons learned from this vulnerability. Any
-    emerging themes?
-  commits:
-  - commit:
-    note:
-  - commit:
-    note:
-i18n:
+      Did the feature that this vulnerability affected use inter-process
+      communication? IPC includes OS signals, pipes, stdin/stdout, message
+      passing, and clipboard. Writing to files that another program in this
+      software system reads is another form of IPC.
+      
+      Answer should be boolean.
+lessons: 
+  complex_inputs: 
+    applies: ~
+    note: ~
+  defense_in_depth: 
+    applies: ~
+    note: ~
+  distrust_input: 
+    applies: true
+    note: "the vulnerability is located in an untrustworthy header.\n"
```

Comment:
Is the _header_ untrustworthy, or is it just that the header can have arbitrary input, of which cannot be trusted?